utils.py

```diff
@@ -154,13 +146,15 @@ def add_component_to_node(gltf2_object, dep, value, export_settings):
     else:
         extensions = gltf2_object.extensions
 
-    existed = True
     if extensions is None:
-        existed = False
         extensions = {}
 
     if hubs_ext_name not in extensions:
         extensions[hubs_ext_name] = {dep.get_name(): value}
+        if type(gltf2_object) is tuple:
+            gltf2_object[0].extensions = extensions
+        else:
+            gltf2_object.extensions = extensions
```

Comment:
I think this is fine, but it seems kind of hard to reason about to me.
Are you sure this shouldn't be up under the `if extensions is None:` block?